lib/constants.rb

```diff
@@ -50,13 +50,14 @@ module Constants
   ].freeze
 
   CONTENT_TYPES = {
-    'image' => Cocina::Models::Vocab.image,
-    'book' => Cocina::Models::Vocab.book,
+    'book (ltr)' => Cocina::Models::Vocab.book,
+    'book (rtl)' => Cocina::Models::Vocab.book,
```

Comment:
add reading direction options for the books, and reorder this list to match what is shown in the registration options

Reply:
I don't think you can do this because there are other places in the code where we call `CONTENT_TYPES#key` and now the values are no longer unique.